### ISSUE TITLE
Material icons all look the same

### ISSUE THREAD

**Issue by Darthagnon** [aardappel/treesheets#623]
For anyone else looking. [7672978988 (26 Jan 2024)](https://github.com/aardappel/treesheets/releases/tag/7672978988) is the last version of Treesheets before the toolbar icons were changed from WinXP/Aero icons to Google Material-style icons. 

Maybe it's just me, but I find the new monochrome Material icons rather ugly and difficult to tell apart.

**Comment by Darthagnon** [comment]
As per [the changelog of 7717149563](https://github.com/aardappel/treesheets/releases/tag/7717149563), documented accessibility regression:

```
Replace toolbar icons with scalable images from Material Design (#605)

The SVG files are rendered into bitmap bundles that can be used for
HiDPI display.

This commit also removes the extra icon set for black and white and its
configuration. The Material Design symbols are already monochrome.
```

The previous toolbar icons seem to work just fine with my HiDPI display, are much easier to tell apart, and look pretty, with a stronger visual identity. They do still disappear offscreen when the window is resized (#247) without any collapsible or scrollable menu, but look and work fine in fullscreen.

**Comment by aardappel** [comment]
Hmm, maybe we can find a really good color icon set that comes in SVG format?

In fact there apparently exist SVG versions of the original icons we used: https://en.wikipedia.org/wiki/Nuvola

**Comment by georgeraraujo** [comment]
> Hmm, maybe we can find a really good color icon set that comes in SVG format?
> 
> In fact there apparently exist SVG versions of the original icons we used: https://en.wikipedia.org/wiki/Nuvola

I tried my hand at those SVG versions, cobbled together new icons, and came up with this:

![image](https://github.com/aardappel/treesheets/assets/4930759/64d316b2-1ceb-4572-9cf5-2faaf64081da)

What do you think?

**Comment by aardappel** [comment]
hard to tell, the rendering of these doesn't look particularly precise pixel wise, but it could just be the screenshot. Are these directly rendered from an SVG without bitmap scaling?

These icons are prettier than the b&w ones, but b&w looks less intrusive.

@tobiolo

**Comment by georgeraraujo** [comment]
> hard to tell, the rendering of these doesn't look particularly precise pixel wise, but it could just be the screenshot. Are these directly rendered from an SVG without bitmap scaling?

Yes, they are. I suppose GitHub lowered the quality of the screenshot to save space.

> These icons are prettier than the b&w ones, but b&w looks less intrusive.

I like the monochrome icons better than these, too.
